### PR TITLE
Fix null constraint bug on created_at and updated_at columns in latest migration

### PR DESCRIPTION
**Pull request by evmiguel**
I was able to reproduce this bug by populating my database with the staging database data:
```
# On staging server
ssh root@aria-at-staging.w3.org
pg_dump -U atr -h localhost aria_at_report > dump.sql

# On local machine
scp root@aria-at-staging.w3.org:dump.sql .
psql -d aria_at_report < dump.sql
```

## Changes in this PR:

- Changed `createdAt` and `updatedAt` to `created_at` and `updated_at` in `run` table
- Set a default value in the migration, which causes Sequelize to auto-populate the rows with null in those columns
- Updated Run model to include the `createdAt` and `updatedAt` attributes.

## Updating database
```
# Rollback to the migration before the latest one
yarn sequelize db:migrate:undo

# Run the migrations
yarn sequelize db:migrate

## This is outdated. There will be nulls.
# Verify that the rows were auto-populated with -infinity
psql -d aria_at_report -c "select id, created_at, updated_at from run;"
```

### CODE REVIEW

File: server/migrations/20200915203422-AddColumnsToTestRun.js [hunk 1]
```diff
@@ -26,19 +26,21 @@ module.exports = {
                 ),
                 queryInterface.addColumn(
                     'run',
-                    'createdAt',
+                    'created_at',
```
Comment (evmiguel) [inline server/migrations/20200915203422-AddColumnsToTestRun.js:29]: The queryInterface does not have the capability to use `underscored: true`

File: server/migrations/20200915203422-AddColumnsToTestRun.js [hunk 2]
```diff
@@ -26,19 +26,21 @@ module.exports = {
...
                     {
                         type: Sequelize.DATE,
-                        allowNull: false
+                        allowNull: false,
+                        defaultValue: Number.NEGATIVE_INFINITY
```
Comment (spectranaut) [inline server/migrations/20200915203422-AddColumnsToTestRun.js:33]: unfortunately, new entries now have negative infinity for this value as well!

**Review by spectranaut — CHANGES_REQUESTED** [review]

File: server/migrations/20200915203422-AddColumnsToTestRun.js (hunk 2 above)
Comment (evmiguel) [reply server/migrations/20200915203422-AddColumnsToTestRun.js:33]: @spectranaut oh dear! Can you show me how you tested what happens with new entries?

Comment (spectranaut) [reply server/migrations/20200915203422-AddColumnsToTestRun.js:33]: yes def! I test by running the migration, then running "yarn dev", then adding a test cycle through the front end, then inspecting the `run` table in the database.

Comment (spectranaut) [reply server/migrations/20200915203422-AddColumnsToTestRun.js:33]: I just tested out of curiosity, changing "defaultValue" to "Sequelize.Now" didn't work --- you still got the null constraints on the migration. I think we have no choice but to allow null. However, I think we also need to add Sequelize.Now to get those values populated. This means, in the migration:
```
                queryInterface.addColumn(
                    'run',
                    'created_at',
                    {
                        type: Sequelize.DATE,
                        defaultValue: Sequelize.NOW
                    },
                    { transaction: t }
                ),
                queryInterface.addColumn(
                    'run',
                    'updated_at',
                    {
                        type: Sequelize.DATE,
                        defaultValue: Sequelize.NOW
                    },
                    { transaction: t }
                )
```
and in the model:
```
            createdAt: {
                type: DataTypes.DATE,
                allowNull: true,
                defaultValue: sequelize.NOW
            },
            updatedAt: {
                type: DataTypes.DATE,
                allowNull: true,
                defaultValue: sequelize.NOW
            }
```

**Comment by spectranaut** [comment]
@evmiguel I just pushed the change I found worked can you run it to test it too? If you're good with it we can merge :)

File: server/migrations/20200915203422-AddColumnsToTestRun.js (hunk 2 above)
Comment (evmiguel) [reply server/migrations/20200915203422-AddColumnsToTestRun.js]: @spectranaut You commit works! I suppose having nulls is just as informative as having something like a negative infinity take the place of a null.

**Review by spectranaut — APPROVED** [review]

**Pull request merged**